op-guide/binary-deployment.md

```diff
@@ -0,0 +1,486 @@
+
+### Database running user
+
+| Configuration | Description |
+| :-- | :---------------------------- |
+| LANG environment | Set `LANG = en_US.UTF8` |
+| TZ time zone | Set the TZ time zone of all nodes to the same value |
+
+## Create the database running user account
+
+In the Linux environment, create TiDB on each installation node as a database running user, and set up the SSH mutual trust between cluster nodes. To create a running user and open SSH mutual trust, contact the system administrator. Here is an example:
+
+```bash
+$ useradd tidb
+$ usermod -a -G tidb tidb
+$ su - tidb
+Last login: Tue Aug 22 12:06:23 CST 2017 on pts/2
+-bash-4.2$ ssh-keygen -t rsa
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/tidb/.ssh/id_rsa):
+Created directory '/home/tidb/.ssh'.
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in /home/tidb/.ssh/id_rsa.
+Your public key has been saved in /home/tidb/.ssh/id_rsa.pub.
+The key fingerprint is:
+5a:00:e6:df:9e:40:25:2c:2d:e2:6e:ee:74:c6:c3:c1 tidb@t001
+The key's randomart image is:
++--[ RSA 2048]----+
+|    oo. .        |
+|  .oo.oo         |
+| . ..oo          |
+|  .. o o         |
+| .  E o S        |
+|  oo . = .       |
+| o. * . o        |
+| ..o .           |
+| ..              |
++-----------------+
+
+-bash-4.2$ cd .ssh
+-bash-4.2$ cat id_rsa.pub >> authorized_keys
+-bash-4.2$ chmod 644 authorized_keys
+-bash-4.2$ ssh-copy-id -i ~/.ssh/id_rsa.pub 192.168.1.100
+```
+
+## Download the official binary package
+
+TiDB provides the official binary installation package that supports Linux. For the operating system, it is recommended to use Redhat 7.3+, CentOS 7.3+ and higher versions.
+
+### Operating system: Linux (Redhat 7+, CentOS 7+)
+
+```
+# Download the package.
+$ wget http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz
+$ wget http://download.pingcap.org/tidb-latest-linux-amd64.sha256
+
+# Check the file integrity. If the result is OK, the file is correct.
+$ sha256sum -c tidb-latest-linux-amd64.sha256
+
+# Extract the package.
+$ tar -xzf tidb-latest-linux-amd64.tar.gz
+$ cd tidb-latest-linux-amd64
+```
+
+## Single node cluster deployment
+
+After downloading the TiDB binary package, you can run and test the TiDB cluster on a standalone server. Follow the steps below to start PD, TiKV and TiDB:
+
+1. Start PD.
+
+    ```bash
+    $ ./bin/pd-server --data-dir=pd \
+                    --log-file=pd.log &
+    ```
+
+
+2. Start TiKV.
+
+    ```bash
+    $ ./bin/tikv-server --pd="127.0.0.1:2379" \
+                      --data-dir=tikv \
+                      --log-file=tikv.log &
+    ```
+
+3. Start TiDB.
+
+    ```bash
+    $ ./bin/tidb-server --store=tikv \
+                      --path="127.0.0.1:2379" \
+                      --log-file=tidb.log &
+    ```
+
+4. Use the official MySQL client to connect to TiDB.
```

Comment:
```suggestion
4. Use the MySQL client to connect to TiDB.
```

Official to me denotes the one distributed by Oracle.  It will work with the MariaDB one too.